themes/default/content/blog/meals-for-ukraine/index.md

```diff
@@ -0,0 +1,58 @@
+meta_image: kubecon-ukraine.png
+authors:
+    - wendy-smith
+tags:
+    - community
+    - pulumi-events
+---
+Why not do good while learning about cloud native tech? [Buoyant](https://buoyant.io/), [Civo](https://www.civo.com/), [Cockroach Labs](https://www.cockroachlabs.com/), [Fairwinds](https://www.fairwinds.com/), [Pulumi](https://www.pulumi.com/), and [Jetstack](https://www.jetstack.io/) have teamed up to donate $2 for each virtual booth visitor to [World Central Kitchen](https://wck.org/) (WCK), a non-profit committed to providing meals in response to humanitarian crises.
+
+<!--more-->
+Working with restaurants across Ukraine and in six countries welcoming refugees, WCK is providing hundreds of thousands of daily meals for people in need. WCK serves meals at shelters, hospitals, transit hubs, and more. It's a wonderful mission and, if you agree, we encourage you to [donate to WCK directly](https://donate.wck.org/give/398293#!/donation/checkout) 💙 🇺🇦 💛
+
+## Tweets
+
+{{< tweet id="1510622155775619078" user="pulumipus" >}}
+
+## How it works
+
+Participating is easy. If you registered for KubeCon, just click on the booth links below to visit our virtual booths. Each visit will count as a $2 donation, so if you visit us all, that's $12 for the cause right there!
+
+Of course, there are many more reasons to visit our booths. You'll find lots of great cloud native resources including demos, tutorials, ebooks, and more. Here are some of the highlights you can expect at each booth.
+
+### Pulumi — Universal Infrastructure as Code
+
+At [Pulumi](https://www.pulumi.com/)’s booth, you'll learn about their cloud engineering platform that brings infrastructure, developer, and security teams together through a unified software engineering process that tames cloud complexity and accelerates innovation. Don't miss Pulumi's Ask an Expert live sessions to hear directly from the engineering team. Whether you want to know more about how to build with Universal Infrastructure as Code in general-purpose programming and markup languages or just learn more about what's on the roadmap, this is your chance to ask anything. [Visit Pulumi today](https://bit.ly/3M5b6Qj)!
+
+### Buoyant — the creator of Linkerd
+
+[Buoyant](https://buoyant.io/) is announcing something really exciting: the introduction of *fully managed Linkerd*. You can be one of the first to check out the demo, and you'll also find a ton of service mesh content including a service mesh 101 in English and Spanish , our engineer's guide to mTLS, and details on Buoyant's Service Mesh Academy. For this and more, visit [the Buoyant booth](https://kubecon-cloudnativecon-europe.com/virtual-exhibitor/?v4b4342b0f72f3260e37d74de68eab433fee0c641d933e76d52be7eb34b211371656f07b5a54b2e522db3ac7b27c7d555=908815D44FE61A2E80915ACEC6E64FC9A625809715300E330E9CA19100987FDC5540D5D12FFF991541BCDBA641CF8FDC)!
```

Comment:
The "engineer's guide" sounds like it might be a title of something -- is it? (Right now it reads a bit like "a guide one of our engineers wrote on mTLS".)

```suggestion
[Buoyant](https://buoyant.io/) is announcing something really exciting: the introduction of *fully managed Linkerd*. You can be one of the first to check out the demo, and you'll also find a ton of service mesh content including a service mesh 101 in English and Spanish, their Engineer's Guide to mTLS, and details on Buoyant's Service Mesh Academy. For this and more, visit [the Buoyant booth](https://kubecon-cloudnativecon-europe.com/virtual-exhibitor/?v4b4342b0f72f3260e37d74de68eab433fee0c641d933e76d52be7eb34b211371656f07b5a54b2e522db3ac7b27c7d555=908815D44FE61A2E80915ACEC6E64FC9A625809715300E330E9CA19100987FDC5540D5D12FFF991541BCDBA641CF8FDC)!
```

>our engineer's guide

Ours, or theirs?

Reply:
Basically the words for each vendor are written by that vendor so we shouldn’t change them- and each of these vendors is doing a similar post with the same format and i think @wamazonsmith-web wrote the collaboratively with them

Reply:
But yeah it should be third party voice